app/assets/stylesheets/application.scss

```diff
@@ -10,5 +10,42 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- */
-@import 'bootstrap';
```

Comment:
this was `application.css`, which I've removed and replaced with `application.scss`